README.md

```diff
@@ -0,0 +1,4 @@
+#README.md
+
```

Comment:
This will override our readme.txt.
We should move readme.txt into README.md in another CL.

Reply:
Take from my README.md anything you find useful. I don't care about authorship.